libs/base/ui/src/utilities/model/common.ts

```diff
@@ -1,7 +1,9 @@
-export enum Size {
-  large = 'large',
-  medium = 'medium',
-  small = 'small',
+export const enum Size {
+  Xs = 'xs',
+  Sm = 'sm',
+  Md = 'md',
+  Lg = 'lg',
+  Xl = 'xl',
 }
```

Comment:
I would say we should add one more reusable variable. (e.g) MainSize = [Size.Lg, Size.Md, Size.Sm] and reuse it in all cases where we need `[Size.Lg, Size.Md, Size.Sm]`